pages/about.html

```diff
@@ -1,12 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <!--- Meta Tags-->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dana Rocha</title>
+
+    <!-- FontAwesome -->
+    <script src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;1,400;1,500&family=Noto+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+
+    <!-- Stylesheet-->
+    <link href="/stylesheets/style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+
+  <body>
+    <main class="container">
+      <!-- Header -->
+      <header>
+        <!-- Navigation -->
+        <nav class="navbar navbar-light bg-light topnav">
+          <h3 class="topnav-left"> 
+            <a href="index.html" class="home-button"> DANA ROCHA </a>
+          </h3>
+
+          <h3 class="topnav-right">
+            <a href="/pages/about.html" class="about-button"> ABOUT </a>
+            <a href="/resume/DanaRocha-Resume.pdf" target="_blank" class="resume-button"> RESUME </a>
+            <a href="/pages/portfolio.html" class="portfolio-button"> PORTFOLIO </a>
```

Comment:
`about.html`, `portfolio.html`, and `index.html` are in the same directory, so you can link them just as `about.html`, `portfolio.html`, and `index.html`(remove `/pages/`)